m3/demo/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM nvidia/cuda:12.2.2-devel-ubuntu22.04
+
+RUN apt-get update
+RUN apt-get install -y wget python3.10 python3.10-venv python3.10-dev git unzip 
+RUN apt-get install -y python3-pip
+
+RUN git clone https://github.com/Project-MONAI/VLM --recursive
+WORKDIR /VLM
+SHELL ["/bin/bash", "-c"] 
+RUN python3.10 -m venv .venv
```

Comment:
Not sure if the virtual environment is required but I followed your instructions @mingxin-zheng

Reply:
Hi @holgerroth , virtual environment is not needed, but you'll need:
```
RUN ln -s /usr/bin/python3.10 /usr/bin/python3
RUN ln -s /usr/bin/python3.10 /usr/bin/python
```

Reply:
Ok. I simplified the dockerfile, removing the venv.